ci/pipeline.yml

```diff
@@ -2,12 +2,12 @@
 ############################
 #  SHARED
 
-cf-staging: &cf-staging
+env-cf: &env-cf
   CF_API: https://api.fr.cloud.gov
-  CF_USERNAME: ((staging-cf-username))
-  CF_PASSWORD: ((staging-cf-password))
+  CF_USERNAME: ((((deploy-env))-cf-username))
+  CF_PASSWORD: ((((deploy-env))-cf-password))
   CF_ORG: gsa-18f-federalist
-  CF_SPACE: staging
+  CF_SPACE: ((deploy-env))
 
 cf-production: &cf-production
```

Comment:
You can delete the shared config for production and any job that uses it since it's going to come from the `instance-var` of `deploy-env`